ecr/Makefile

```diff
@@ -0,0 +1,31 @@
+all: init plan apply aws build tag push
+
+#Terraform
+
+init: 
+	terraform init
+
+plan: init
+	terraform plan 
```

Comment:
change to `terraform plan -out=.terrafom-plan`

Reply:
also included the '-var-file terraform.tfvars' to refer to the variables file